profiler/internal/pprofutils/delta_test.go

```diff
@@ -0,0 +1,94 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package pprofutils
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/matryer/is"
+)
+
+func TestDelta(t *testing.T) {
+	t.Run("simple", func(t *testing.T) {
+		var (
+			is        = is.New(t)
+			deltaText bytes.Buffer
+		)
+
+		profA, err := Text{}.Convert(strings.NewReader(strings.TrimSpace(`
+main;foo 5
+main;foo;bar 3
+main;foobar 4
+`)))
+		is.NoErr(err)
+
+		profB, err := Text{}.Convert(strings.NewReader(strings.TrimSpace(`
+main;foo 8
+main;foo;bar 3
+main;foobar 5
+`)))
+		is.NoErr(err)
+
+		delta, err := Delta{}.Convert(profA, profB)
+		is.NoErr(err)
+
+		is.NoErr(Protobuf{}.Convert(delta, &deltaText))
+		is.Equal(deltaText.String(), strings.TrimSpace(`
+main;foo 3
+main;foobar 1
+`)+"\n")
+	})
+
+	t.Run("sample types", func(t *testing.T) {
```

Comment:
I think we should use just one word everywhere if possible, in sub-test names. We've been using this convention throughout. Now that you've copied the code over, it might make sense to slightly change things to make it seem as if its part of this repo (which it is now)

Reply:
f282778782da17982874d4c95610524e9d9a3f34